core/src/epicli/data/common/validation/configuration/backup.yml

```
@@ -17,3 +17,66 @@ properties:
     properties:
       components:
         type: object
+        additionalProperties: false
+        properties:
+          load_balancer:
+            "$id": "#/properties/specification/properties/components/properties/load_balancer"
+            type: object
+            required:
+            - enabled
+            additionalProperties: false
+            properties:
+              enabled:
+                "$id": "#/properties/specification/properties/components/properties/load_balancer/properties/enabled"
+                type: boolean
+          logging:
+            "$id": "#/properties/specification/properties/components/properties/logging"
+            type: object
+            required:
+            - enabled
+            additionalProperties: true
```

Looks like an overlook.
```suggestion
            additionalProperties: false
```

---

Fix in 84c68b8